catalyst-gateway/bin/src/service/api/fragments/mod.rs

```diff
@@ -0,0 +1,40 @@
+//! Fragment endpoints
```

Comment:
Can you please add these endpoints to the `V1Api`?

Reply:
I was taking a look at the code. Does moving it there mean we wouldn't be able to use different tags for grouping related endpoints?

Reply:
Yeah, I've the same question. Perhaps @stevenj can clarify how this should best be handled.

Reply:
Actually not, you can add a specific tag for each endpoint separately, e.g.:
```Rust
#[oai(path = "/", method = "post", operation_id = "fragments", tag = "ApiTags::Fragments")]
    async fn index(
        &self,
        /// Batch of fragments to be processed.
        Json(fragments_batch): Json<FragmentsBatch>,
    ) -> index::AllResponses
  ```